### DISCUSSION TITLE
p.48 COLUMN「『internal』パッケージ」内の `Hooga`/`Hoge` 誤植っぽさ

### DISCUSSION THREAD

**Discussion by yakkun** [budougumi0617/go_todo_app#80] [Ideas]
はろー。おひさしぶりでーす。
人に読ませようと思って改めて読んでたら気付きました。誰もまだ指摘してなさそうだったので。

<img width="527" alt="image" src="https://user-images.githubusercontent.com/696261/213642214-47735701-38f1-489a-af11-0197207c1934.png">
P.48 より (Kindle 版の 2022年8月1日初版発行 Ver1.0)

後半の段落で出てくる `Hooga` 型と、そのあとでアクセスできない `Hoge` 型というのが同じ型名であるべきっぽいですが、あってなさそうでした！

**Comment by budougumi0617** [comment]
ご無沙汰してます！
紹介に加えて指摘までしていただけるとは大変ありがたいです！正誤表に追加しておきました！
- https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.12
